src/components/Heading/Heading.module.scss

```diff
@@ -27,63 +28,6 @@
 
 /* Override tags styles */
 
-.elementTypeH1 {
-  letter-spacing: -0.2px;
-  font-size: $h1-legacy-font-size;
-  line-height: $h1-legacy-line-height;
-  font-weight: var(--font-weight-bold);
-  font-family: var(--title-font-family);
-  -webkit-font-smoothing: var(--font-smoothing-webkit);
-  -moz-osx-font-smoothing: var(--font-smoothing-moz);
-}
-
-.elementTypeH1.sizeSmall,
-.elementTypeH1.sizeMedium {
-  font-size: var(--font-size-h2);
-  line-height: var(--font-line-height-h2);
-}
-
-.elementTypeH2 {
-  letter-spacing: -0.2px;
-  @include font-title();
-}
-
-.elementTypeH2.sizeSmall,
-.elementTypeH2.sizeMedium {
-  @include font-subtitle();
-}
-
-.elementTypeH3 {
-  letter-spacing: -0.2px;
-  @include font-title-secondary();
-}
-
-.elementTypeH3.sizeSmall,
-.elementTypeH3.sizeMedium {
-  @include font-subtitle();
-}
-
-.elementTypeH4 {
-  letter-spacing: -0.2px;
-  @include font-subtitle();
-}
-
-.elementTypeH5 {
-  @include font-paragraph-bold();
-}
-
-.elementTypeH6 {
-  @include font-general-text();
-}
-
-.elementTypeP {
-  @include font-paragraph();
-}
-
-.elementTypeSmall {
-  @include font-caption();
-}
-
 .elementTypeLink {
```

Comment:
what about this one?

Reply:
I removed all font detentions and kept only link appearance related styles (which are not part of the class currently...)

Not sure if we want to define color related styles in the class